### PR TITLE
Update UI-Kit to v1.7.5

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -143,6 +143,15 @@
...
 
+  create_table "sessions", force: :cascade do |t|
+    t.string   "session_id", null: false
+    t.text     "data"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.index ["session_id"], name: "index_sessions_on_session_id", unique: true, using: :btree
+    t.index ["updated_at"], name: "index_sessions_on_updated_at", using: :btree
+  end
+
```
Comment (ltankey) [inline db/schema.rb]: Is this supposed to be part of this PR?

**Comment by ltankey** [comment]
LGTM

**Comment by bjgordon** [comment]
LGTM

**Comment by maxious** [comment]
rebased

**Pull request merged**